src/apiCalls.js

```diff
@@ -33,7 +33,7 @@ export let apiCalls = {
   },
 
   deleteBooking(bookingToDelete) {
-    fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings', {
+    return fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings', {
```

Comment:
Good catch on adding the return statement when using a DELETE with the fetch API.